pset-functions.js

```diff
@@ -136,6 +182,32 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let letters = {
```

Comment:
@rvaswani85 nice approach of using an object to represent the letter / score combinations